src/systems/filecoin_blockchain/struct/block/election.go

```diff
@@ -3,7 +3,7 @@ package block
 import (
 	addr "github.com/filecoin-project/go-address"
 	abi "github.com/filecoin-project/specs-actors/actors/abi"
-	crypto "github.com/filecoin-project/specs/algorithms/randomness"
```

Comment:
I am not sure what `randomness` package is referring to, because I can't find such package.

Reply:
This change is problematic because the actors/crypto is implemented from the spec. This spec (or discussion/issues etc) is how we know what to write. It would really be much better if the spec duplicated what it needs from actors/crypto rather than import it. Otherwise, if we change actors, we'll have no way to know that we messed it up.

Reply:
I am not sure I understand how to solve this.

In this repo `github.com/filecoin-project/specs-actors/actors` is replaced with `replace github.com/filecoin-project/specs-actors => ../../src/actors`.

So `github.com/filecoin-project/spec-actors` is already duplicated/vendored (and actually differs, for example `DeriveRandWithMinerAddr` is present locally, and not part of `github.com/filecoin-project/spec-actors` AFAICT).